src/data-input.js

```diff
@@ -0,0 +1,26 @@
+import React, {PropTypes as t} from 'react';
+
+const DataInput = ({autoFocus, name, onChange, onKeyPress, value, label}) => (
```

Comment:
I like to alphabetize the destructured props since there is no other logical ordering.